### PR TITLE
Russian Inspiration

### PR DESCRIPTION
**Pull request by candicecz**
### New Stylesheet Submission:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders an unminified, compiled CSS file?
- [x] Do your styles pass proper contrast for accessibility?
- [n/a] Have you included `@media (prefers-reduced-motion: reduce)` to remove animations on `*,*::before,*::after`?
- [x] Did you style the `.skip-link`?
- [x] Have you checked on a real mobile device to ensure your responsive design doesn't cause overlaps or overflow scrolling?

### CODE REVIEW

**Comment by candicecz** [comment]
Will need to run the tests again I think, as I closed the PR and reopened it after making many changes. Thanks!

File: src/_data/styles/russian-inpiration [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  // Required
```
Comment (5t3ph) [inline src/_data/styles/russian-inpiration:2]:
```suggestion
```

File: src/_data/styles/russian-inpiration [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "title": "Russian Inspiration",
+  "author": "CandiceCz",
+  "stylesheet": "https://github.com/candicecz/stylestage-sass/blob/main/public/style.css",
+  // Optional
```
Comment (5t3ph) [inline src/_data/styles/russian-inpiration:6]:
```suggestion
```

File: src/_data/styles/russian-inpiration [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "twitter": "tw33t",
```
Comment (5t3ph) [inline src/_data/styles/russian-inpiration:7]: Please don't use a fake username, this is not a required field.

```suggestion
```

**Review by 5t3ph — CHANGES_REQUESTED** [review]

**Comment by 5t3ph** [comment]
Hi @candicecz - the filename needs to be updated to `russian-inspiration.json` to be correctly identified. Also the comments need removed in that file, and please do not enter a wrong Twitter username, it is not a required field. Thanks!

**Comment by candicecz** [comment]
My bad! I changed added the file extension. Thanks for making the other changes.

File: src/_data/styles/russian-inpiration.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Russian Inspiration",
+  "author": "CandiceCz",
+  "stylesheet": "https://github.com/candicecz/stylestage-sass/blob/main/public/style.css",
```
Comment (5t3ph) [inline src/_data/styles/russian-inpiration.json:4]:
```suggestion
  "stylesheet": "https://raw.githubusercontent.com/candicecz/stylestage-sass/main/public/style.css",
```

**Comment by 5t3ph** [comment]
Hi @candicecz - the preview is now available: https://deploy-preview-74--moderncss-styles.netlify.app/styles/russian-inspiration/

The only issue is that the `.skip-link` isn't revealed on keyboard interaction, which can be resolved by updating the selector to include `:focus` in addition to the existing `:hover` rule 👍 thanks!

**Comment by candicecz** [comment]
Hi @5t3ph , good call -I made the change. Thanks!

**Review by 5t3ph — APPROVED** [review]
@candicecz great job! I especially love the nesting dolls :)

Thank you for participating! If you haven't already, consider subscribing for updates to learn about new styles and features: https://stylestage.dev/subscribe/

**Pull request merged**